lua/dylib.lua

```diff
@@ -0,0 +1,20 @@
+module(..., package.seeall)
+
+--
+-- C functions exported through FFI
+--
+
+local ffi = require("ffi")
+
+-- Structs
+ffi.cdef[[
+
+extern const char *CLIENT_PROC_ERROR;
```

Comment:
The Lua requests don't need this constant since this string will only show up if something unexpected happens. Moreover, the process talking to Gatekeeper is the one that will receive this string, not the Lua request.